google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/resources/ProjectSelector.java

```diff
@@ -391,6 +393,25 @@ public void showPopup(RelativePoint showTarget) {
     }
   }
 
+  @Override
+  public void addFocusListener() {
```

Comment:
Small nit, but I'd argue that it might be cleaner to just call getTextField.addFocusListener(...) directly in this class's constructor rather than overriding this method. That way this class has control of how/when the listener is added.